README.md

```diff
@@ -1,11 +1,18 @@
 # DEV Stack
 Docker-compose stack of all essential blockr modules
 
-## Steps
+## Steps run blockcahin for the first time
  - Clone project
+ - Change .gitmodules branch of validator to `Develop`
  - Run `git submodule init`
  - Run `git submodule update --remote`
  - Run `docker-compose up --build`
+ - Add `PUBKEY_ADMIN` and `PRIVKEY_ADMIN` from `/keys/.keys` generated by the validator_backup to `blockr-validator/src/.env`
```

Comment:
This is not clear for the end-user. Describe how they can access the shell of the container.